### PR TITLE
Space Travelers Project

### PR DESCRIPTION
**Pull request by blakbox23**
In this project we built  a space project with react and redux. Tasks include: 
   - Config & basic setup
   - Redux: Fetch data and update Redux store
   - Render UI: lists
   - Redux: Write actions and reducers for booking rockets/dragons and joining missions
   - Redux: Write actions and reducers for canceling rockets/dragons and leaving missions
   - Render UI: conditional components rendering
   - Render UI: My Profile section

### CODE REVIEW

File: src/components/MissionBookingUI.JS [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+//         )}
+//           {reserved && (
+//             <Button variant="outline-dark">Leave Mission</Button>
+//         )}
+        
+//         </>
+//     )
+// }
+
+// export default MissionBookingUI
```
Comment (Emmyn5600) [inline src/components/MissionBookingUI.JS]: - [x] Remove the commented code in the project that has been pushed

File: src/components/Missions.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+import { Table } from 'react-bootstrap';
+import Badge from 'react-bootstrap/Badge';
+import Button from 'react-bootstrap/Button';
+import { getMissions } from '../redux/api';
+import { joinMission, leaveMission } from '../redux/missions/MissionsStore';
+// import MissionBookingUI from './MissionBookingUI.JS';
+
+function Missions() {
+  const missions = useSelector((state) => state.missionsState);
+  const dispatch = useDispatch();
```
Comment (Emmyn5600) [inline src/components/Missions.js]: - [x] Let us use ES6 syntax.

File: src/components/Profile.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import { Table } from 'react-bootstrap';
+
+function Profile() {
+  const rockets = useSelector((state) => state.rocketsState);
+  const allRockets = rockets.filter((rocket) => rocket.reserved);
```
Comment (Emmyn5600) [inline src/components/Profile.js]: - [x] Let us use ES6 syntax.

File: src/components/Rockets.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Badge from 'react-bootstrap/Badge';
+import Button from 'react-bootstrap/Button';
+import { getRockets } from '../redux/api';
+import { joinRocket, leaveRocket } from '../redux/rockets/RocketsStore';
+
+function Rockets() {
+  const dispatch = useDispatch();
+  const rockets = useSelector((state) => state.rocketsState);
```
Comment (Emmyn5600) [inline src/components/Rockets.js]: - [x] Let us use ES6 syntax.

File: src/components/Missionitem.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+/* eslint react/prop-types: 0 */
+
+import React from 'react';
+
+function Missionitem({ mission }) {
+  return (
+    <div>
+      <p>{mission.mission_name}</p>
+    </div>
+  );
```
Comment (Emmyn5600) [inline src/components/Missionitem.js]: - [x] Let us use ES6 syntax.

File: src/components/Missions.js [hunk 6]
```diff
@@ -0,0 +1,74 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Table } from 'react-bootstrap';
+import Badge from 'react-bootstrap/Badge';
+import Button from 'react-bootstrap/Button';
+import { getMissions } from '../redux/api';
+import { joinMission, leaveMission } from '../redux/missions/MissionsStore';
+// import MissionBookingUI from './MissionBookingUI.JS';
+
```
Comment (Emmyn5600) [inline src/components/Missions.js]: - [x] Remove the commented line

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

### Hi, @blakbox23 and @blakbox23 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:
   
 - The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- You have descriptive Readme file:heavy_check_mark:
- All tests are passing successfully :heavy_check_mark:
- The live link demo is working properly :heavy_check_mark:

   ###  Changes required :recycle: 
   
     - _Check the comment under the review_
    
    Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by Davidosky007** [comment]
## Hi @blakbox23 @igorkol91 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

nice-job


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**